vars/folioHelmFlow.groovy

```diff
@@ -4,6 +4,15 @@ import org.folio.models.RancherNamespace
 
 import java.time.LocalDateTime
 
+void deployEureka(String namespace) {
```

Comment:
Another portion of unrelated lines...

Reply:
Deployment steps are removed until it will be clear what to put here.